tests/fv3_conf/cpld_control_run.IN

```diff
@@ -42,6 +42,7 @@ cp    @[INPUTDATA_ROOT]/${FV3_DIR}/INPUT/grid_spec.nc ./INPUT/@[ATMRES]_mosaic.n
 
 # MOM6 fixed input
 cp    @[INPUTDATA_ROOT]/MOM6_FIX/@[OCNRES]/* ./INPUT
+cp /scratch1/NCEPDEV/global/glopara/fix/mom6/20220805/500/oceanda_zgrid_25L.nc ./INPUT
```

Comment:
@jkbk2004 has the input files been put on all the platforms?  If so I can revert this and re-run tests as needed.

Reply:
to input-data-20221101/MOM6_FIX/500/oceanda_zgrid_25L.nc, right?

Reply:
rsynced ok input-data-20221101/MOM6_FIX/500/oceanda_zgrid_25L.nc on hera/orion/hercules/gaea/jet. We will clean up cheyenne and replace with Derecho soon.

Reply:
> to input-data-20221101/MOM6_FIX/500/oceanda_zgrid_25L.nc, right?

@jkbk2004  that's correct.   I'll remove this line and re-run on hera.

Reply:
It looks like it'll be this PRs turn soon.  I did just double check and we're still missing the new input file input-data-20221101/MOM6_FIX/500/oceanda_zgrid_25L.nc on WCOSS2.

Reply:
@JessicaMeixner-NOAA: Done.
```
emc.nems@clogin01:/lfs/h2/emc/nems/noscrub/emc.nems/RT/NEMSfv3gfs/input-data-20221101/MOM6_FIX/500> ls
hycom1_25.nc      MOM_override          ocean_hgrid.nc  ocean_mosaic.nc
layer_coord25.nc  oceanda_zgrid_25L.nc  ocean_mask.nc   ocean_topog.nc
```